src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt

```diff
@@ -38,22 +39,21 @@ class AlertsNomisApiService(@Qualifier("nomisApiWebClient") private val webClien
 
   suspend fun getPrisonerIds(pageNumber: Long, pageSize: Long): RestResponsePage<PrisonerId> = webClient.get()
     .uri {
-      it.path("/prisoners/ids")
+      it.path("/prisoners/ids/all")
         .queryParam("page", pageNumber)
         .queryParam("size", pageSize)
-        .queryParam("active", false)
         .build()
     }
     .retrieve()
     .awaitBody()
 
-  suspend fun getAlertsToMigrate(offenderNo: String): PrisonerAlertsResponse = webClient.get()
```

Comment:
because we now get prisoners with no bookings we have to cope with 404 from this endpoint